lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -1101,6 +1101,29 @@ std::string CreateUniqueVariableNameForDisplay(lldb::SBValue v,
 //                       can use this optional information to present the
 //                       children in a paged UI and fetch them in chunks."
 //     }
+//     "declaration": {
+//       "type": "object | undefined",
```

Comment:
What is "type"? I also don't see you setting it in the diff below this one.

Reply:
this "type" is a JSON schema attribute, not an actual thing being returned